lib/proxy/log_buffer/ring_buffer.rb

```diff
@@ -0,0 +1,56 @@
+module Proxy::LogBuffer
```

Comment:
should you update https://github.com/theforeman/smart-proxy/blob/develop/LICENSE too ?